ddg_api.py

```diff
@@ -43,7 +64,8 @@ def byte_image():
 # Pass R group IDs as queries: /molecule?r1=A01&r2=B10
 @app.route("/molecule")
 def molecule_img():
-    """Returns bytestream image of compund molecule consisting of
+    """Returns bytestream image  and drug properties
+    of compund molecule consisting of
     scaffold and up to two R Groups, if these are specified in
     the query parameters.
     Pass R group IDs as queries: /molecule?r1=A01&r2=B10
```

Comment:
What does /molecule?r1=A01&r2=B10 mean?

Reply:
thats an example url used to trigger that function in the api. anything after ? is part of the query, & separates the parameters from each other